kolibri/core/templates/kolibri/base.html

```diff
@@ -7,6 +7,7 @@
   {% base_frontend_async %}
   {% base_frontend_sync %}
   <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" />
+  <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" />
```

Comment:
@rtibbles I've been keeping this in mind and have been discussing it with the other devs. I'd like to work on it, and can look at the documentation, but I don't think I can implement the entire thing myself.